.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           name: mypy
           command: |
             . venv/bin/activate
-            mypy fonz
+            mypy fonz --ignore-missing-imports
```

Comment:
mypy keeps throwing an import error for a package that is definitely imported and is working as far as I can tell. I confirmed that the package is installed in our CI build so I'm just disabling this feature because I'm not sure what else to do about it.

Reply:
Cool. Happy to disable for the moment. We can always come back to it in the future.